test/components/deprecated-tag.test.ts

```diff
@@ -0,0 +1,47 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import { ComponentDefinition } from '../../src/components/interfaces';
+import { buildProject } from './test-helpers';
+
+let component: ComponentDefinition;
+beforeAll(() => {
+  const result = buildProject('deprecated-tag');
+  expect(result).toHaveLength(1);
+
+  component = result[0];
+  console.log(component);
```

Comment:
I'd recommend to remove this `console.log` here.

Reply:
✅